### PR TITLE
Enhancement: Add reviewdog/action-misspell

### LINKED ISSUE #23: Add action-misspell
Add `action-misspell` to correct commonly misspelled English words in source files.

Link:
https://github.com/reviewdog/action-misspell

### PR DESCRIPTION
**Pull request by raydwaipayan**
Add action-misspell to workflows.

Closes #23

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -11,3 +11,14 @@ jobs:
...
+      name: runner / misspell
+      runs-on: ubuntu-latest
+      steps:
+        - name: Check out code.
+          uses: actions/checkout@v1
+        - name: misspell
+          uses: reviewdog/action-misspell@v1
+          with:
+            github_token: ${{ secrets.github_token }}
+            locale: "US"
```
Comment (mgrachev) [inline .github/workflows/reviewdog.yml:24]:
```suggestion
    name: runner / misspell
    runs-on: ubuntu-latest
    steps:
      - name: Check out code.
        uses: actions/checkout@v1
      - name: misspell
        uses: reviewdog/action-misspell@v1
        with:
          github_token: ${{ secrets.github_token }}
          locale: "US"
```

File: .github/workflows/reviewdog.yml [hunk 2]
```diff
@@ -19,4 +30,4 @@ jobs:
       - name: shellcheck
         uses: reviewdog/action-shellcheck@v1
         with:
-          github_token: ${{ secrets.github_token }}
+          github_token: ${{ secrets.github_token }}
```
Comment (mgrachev) [inline .github/workflows/reviewdog.yml:33]: Please add a new line to the end of this file.

Comment (raydwaipayan) [reply .github/workflows/reviewdog.yml]: It's done, thaks for pointing out.

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@raydwaipayan Thanks a lot! 🚀

**Pull request merged**